uktrade/data-hub-frontend
# Feature/investments profiles filters

## Description of change

This PR adds the _Countries of origin_ typeahead filter to the `/investments/profiles` page. The feature is hidden behind the `capital-investments-filters` feature flag.

## Test instructions

1. Enable the `capital-investments-filters` feature flag in Django Admin
2. Go to the `/investments/profiles` page
3. There should be a sidebar with the _Countries of origin_ typeahead filter
4. Select one or multiple countries with the filter
5. The number of results should update and reflect the selected countries
6. There appear a _chip_ for each selected country in the header of the result list
7. Clicking on a chip should remove its corresponding country from the filter and the results should update
8. If the `capital-investments-filters` feature flag is disabled, the page should work as before this PR, i.e. should have no filter sidebar

## Screenshots
### Before

<img width="677" alt="Screen Shot 2021-02-04 at 5 31 28 PM" src="https://user-images.githubusercontent.com/2333157/106931784-dc68f080-670e-11eb-9c22-cdeb4fc92bfd.png">

### After
<img width="681" alt="Screen Shot 2021-02-04 at 5 30 29 PM" src="https://user-images.githubusercontent.com/2333157/106931785-dd018700-670e-11eb-92ae-6a9ac73a7101.png">



_Add a screenshot_
